Doc/library/tkinter.rst

```diff
@@ -67,16 +67,37 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
-
-
-.. function:: Tcl(screenName=None, baseName=None, className='Tk', useTk=0)
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=True, sync=False, use=None)
+
+   Toplevel Tk widget, which is usually the main window of an application.
+   Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the :envvar:`DISPLAY` environmental variable. (X11 only)
+   *baseName*
+      Reads the profile file :file:`{baseName}.tcl` into the Tcl interpreter
+      and calls :func:`exec` on the contents of :file:`{baseName}.py`.
+   *className*
+      Name of the widget class.  Used as a profile file name
+      (:file:`{className}.tcl` and :file:`{className}.py` load before the
+      *baseName* files), and also used as the name with which Tcl is
+      invoked (*argv0* in *interp*).
+   *useTk*
+      If ``True``, initialize the Tk subsystem.  :func:`Tcl` sets this to
+      ``False``.
+   *sync*
+      If ``True``, execute all X server commands synchronously, so that errors are
+      reported immediately.
+   *use*
+      Specifies that the main window for the application is to be embedded in
+      the window whose identifier is given, instead of being created as an
+      independent toplevel window.
+
```

Comment:
While we are documenting the Tk class. Would it be worth it to document Tk attributes? master, children and tk.

Reply:
The current page reads more like a 'getting started' or 'how to', so I wasn't sure how much detail to add.  I know there's a separate bug issue where Terry figured out the difference between master and parent, so maybe it would be worthwhile to include that here?